script.js

```diff
@@ -0,0 +1,37 @@
+//assign values to numOneEl and numTwoEl and operatorEl
+function defineValues (){
+    let numOneEl = Number(document.getElementById("numOne").value)
```

Comment:
Nice job pairing this down - a lot of folks had two lines here, one to get the element from the document, and another to then get the .value of the element. You did it all in one! Great job.